book.rb

```diff
@@ -1,16 +1,23 @@
 require_relative 'rental'
+require_relative 'data_related'
 
 # rubocop: disable Style/ClassVars
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `rubocop`, it will be nice to fix the issue.